docs/content/docs/user-docs/resource-crud.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Resource Creation"
+description: "Create and Delete an S3 bucket"
+lead: "Create and Delete an S3 bucket using ACK controller"
+draft: false
+menu:
+  docs:
+    parent: "installing"
+weight: 30
+toc: true
+---
+
```

Comment:
I think this document could really do with a description. Perhaps explaining to users:
- This is an example of how to create, update and delete an ACK resource
- We are using a basic S3 bucket as an example, here is a link to the API reference for S3
- For any other controllers, here is a link to the API reference overview page

I don't think the "Verify that S3 bucket was created" is a necessary addition. Adding a `kubectl describe` after the `kubectl create` would suffice.

Adding an example of how to update the spec might be useful, though. You could add a tag to the `spec.tagging.tagSet`?